src/EditorFeatures/TestUtilities2/Utilities/GoToHelpers/MockNavigableItemsPresenter.vb

```diff
@@ -1,27 +1,35 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+Imports System.Composition
 Imports System.Threading
 Imports Microsoft.CodeAnalysis.Editor.FindUsages
 Imports Microsoft.CodeAnalysis.Editor.Host
 Imports Microsoft.CodeAnalysis.FindUsages
+Imports Microsoft.CodeAnalysis.Host.Mef
 
 Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Utilities.GoToHelpers
+    <Export(GetType(IStreamingFindUsagesPresenter))>
+    <[Shared]>
+    <PartNotDiscoverable>
     Friend Class MockStreamingFindUsagesPresenter
         Implements IStreamingFindUsagesPresenter
 
         Public ReadOnly Context As New SimpleFindUsagesContext(CancellationToken.None)
-        Private ReadOnly _action As Action
 
-        Public Sub New(action As Action)
-            _action = action
+        <ImportingConstructor>
+        <Obsolete(MefConstruction.ImportingConstructorMessage, True)>
+        Public Sub New()
         End Sub
 
+        Public Property Action As Action
```

Comment:
I worry if this ever gets assigned more than once, since it means two tests are sharing state in some way there's a potential risk of one depending on the other's value. Worth adding a block?

Reply:
I would have been impossible for this to occur. The MEF catalog is forcibly disposed between tests.